GUI/UI/UIInventory.cpp

```diff
@@ -11,33 +11,37 @@
 #include "GUI/GUIButton.h"
 #include "GUI/UI/UIStatusBar.h"
 
-#include "IO/Keyboard.h"
-#include "IO/Mouse.h"
+#include "Io/Mouse.h"
+#include "Io/KeyboardInputHandler.h"
 
 void GUIWindow_Inventory::Update() {
     DrawMessageBox(0);
     DrawText(pFontLucida, 10, 20, 0, "Making item number", 0, 0, 0);
-    DrawText(pFontLucida, 10, 40, 0, pKeyActionMap->pPressedKeysBuffer, 0, 0, 0);
-    if (!pKeyActionMap->field_204) {
-        ItemGen ItemGen2;
-        ItemGen2.Reset();
-        Release();
-        pEventTimer->Resume();
-        current_screen_type = CURRENT_SCREEN::SCREEN_GAME;
-        viewparams->bRedrawGameUI = 1;
-        int v39 = atoi(pKeyActionMap->pPressedKeysBuffer);
-        if (v39 > 0 && v39 < 800) {
-            SpawnActor(v39);
-        }
-    }
+    DrawText(pFontLucida, 10, 40, 0, keyboardInputHandler->GetTextInput().c_str(), 0, 0, 0);
+
+    // a hack to capture end of user input (enter) while avoiding listening to UI message handler
+    // redo this in a more clean way
+    __debugbreak();
```

Comment:
should we capture as an issue or leave a TODO comment?

Reply:
i feel we should consider deleting this. that was some messy way to debug adding items and spawrning minsters using status bar as a console (my guess). not sure that worked at all in the first place